<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>editAddressee_ui</class>
  <widget class="QWidget" name="editAddressee_ui" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>473</width>
        <height>557</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <property name="resizeMode" >
        <enum>FreeResize</enum>
      </property>
      <item row="1" column="0" >
        <widget class="KLineEdit" name="txt_cname" />
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Contact name</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="textLabel2" >
          <property name="text" >
            <string>Phone numbers</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="KLineEdit" name="txt_number" />
          </item>
          <item>
            <widget class="KComboBox" name="cb_type" />
          </item>
        </layout>
      </item>
      <item row="4" column="0" >
        <layout class="QHBoxLayout" >
          <item>
            <widget class="K3ListView" name="lv_numbers" >
              <property name="resizePolicy" >
                <enum>AutoOneFit</enum>
              </property>
              <property name="resizeMode" >
                <enum>LastColumn</enum>
              </property>
              <property name="fullWidth" >
                <bool>true</bool>
              </property>
              <property name="itemsRenameable" >
                <bool>true</bool>
              </property>
              <column>
                <property name="text" >
                  <string>Phone number</string>
                </property>
                <property name="clickable" >
                  <bool>true</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
              <column>
                <property name="text" >
                  <string>Type</string>
                </property>
                <property name="clickable" >
                  <bool>false</bool>
                </property>
                <property name="resizable" >
                  <bool>true</bool>
                </property>
              </column>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout" >
              <item>
                <widget class="KPushButton" name="b_add" >
                  <property name="text" >
                    <string>&amp;Add</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="KPushButton" name="b_del" >
                  <property name="text" >
                    <string>Remo&amp;ve</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>31</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item row="5" column="0" >
        <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
            <string>Memory slot to use</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="6" column="0" >
        <widget class="KComboBox" name="cb_memslot" />
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
    </customwidget>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >klineedit.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kcombobox.h</include>
    <include location="local" >k3listview.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kpushbutton.h</include>
    <include location="local" >kcombobox.h</include>
  </includes>
</ui>
